File: bookstore-backend/src/main/java/com/bookstore/bookstore/presentation/response/LoginResponse.java
package com.bookstore.bookstore.presentation.response;

import com.bookstore.bookstore.domain.enums.UserStatus;
import java.util.Set;
import java.util.UUID;

public record LoginResponse(
        UUID userId,
        UserStatus status,
        Set<String> roles,
        String accessToken,
        String refreshToken
) {
}

File: bookstore-desktop/Bookstore.Desktop/Dtos/LoginResponse.cs
namespace Bookstore.Desktop.Dtos;

public class LoginResponse
{
    public string UserId { get; init; } = "";
    public string Status { get; init; } = "";
    public IReadOnlyList<string> Roles { get; init; } = Array.Empty<string>();
    public string AccessToken { get; init; } = "";
    public string RefreshToken { get; init; } = "";
}
